core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -413,21 +408,25 @@ class LogCleaner(initialConfig: CleanerConfig,
     def recordStats(id: Int, name: String, from: Long, to: Long, stats: CleanerStats): Unit = {
       this.lastStats = stats
       def mb(bytes: Double) = bytes / (1024*1024)
+      val bytesReadMb = mb(stats.bytesRead.toDouble)
+      val mapBytesReadMb = mb(stats.mapBytesRead.toDouble)
+      val bytesWrittenMb = mb(stats.bytesWritten.toDouble)
+
       val message =
         "%n\tLog cleaner thread %d cleaned log %s (dirty section = [%d, %d])%n".format(id, name, from, to) +
-        "\t%,.1f MB of log processed in %,.1f seconds (%,.1f MB/sec).%n".format(mb(stats.bytesRead.toDouble),
+        "\t%,.1f MB of log processed in %,.1f seconds (%,.1f MB/sec).%n".format(bytesReadMb,
                                                                                 stats.elapsedSecs,
-                                                                                mb(stats.bytesRead.toDouble / stats.elapsedSecs)) +
-        "\tIndexed %,.1f MB in %.1f seconds (%,.1f Mb/sec, %.1f%% of total time)%n".format(mb(stats.mapBytesRead.toDouble),
+                                                                                bytesReadMb / stats.elapsedSecs) +
+        "\tIndexed %,.1f MB in %.1f seconds (%,.1f Mb/sec, %.1f%% of total time)%n".format(mapBytesReadMb,
```

Comment:
No code change, just replace the `mb(stats.bytesRead.toDouble)` as variable `bytesReadMb` and so on.